lab_5_scrapper/scrapper.py

```diff
@@ -127,11 +222,27 @@ def _extract_url(self, article_bs: BeautifulSoup) -> str:
         Returns:
             str: Url from HTML
         """
+        url = ''
+        for div in article_bs.find(class_="card-body").find_all("div", {"class": "title"}):
+            for link in div.select("a"):
+                url = link['href']
+        return self.url_pattern + url
```

Comment:
what is the reason to loop over ALL urls and take only LAST?

Reply:
fixed it